frontend/src/components/statsCardContent.js

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 
 export const StatsCardContent = ({ value, label, className, invertColors = false }: Object) => (
   <div className={className}>
-    <h3 className={`ma0 mb2 barlow-condensed f2 b ${invertColors ? 'white' : 'red'}`}>{value}</h3>
+    <h3 className={`ma0 mb2 barlow-condensed f3 f4-m b ${invertColors ? 'white' : 'red'}`}>
```

Comment:
That component is used in a lot of other pages. It's not an option to reduce the font size.